include/rmm/device_scalar.hpp

```diff
@@ -60,28 +56,86 @@ class device_scalar {
    * @return T The value of the scalar after synchronizing its stream
    *---------------------------------------------------------------------------**/
   T value() const {
```

Comment:
Delete this version, since PR #169 now removes the "stored stream" semantics of `device_buffer`.